_source/_posts/2023-07-27-spring-graphql-react.md

```diff
@@ -1306,6 +1310,12 @@ Again, in the root folder, run the API server with:
 source .env && ./gradlew bootRun
 ```
 
+Get an access token using the Auth0 cli with the `test token` command:
+
+```shell
+auth0 test token -a https://<yourAuth0Domain>/api/v2/
```

Comment:
Is `<yourAuth0Domain>` the same placeholder you've used in other places? It doesn't matter what we use, as long as we're consistent.

Reply:
Yes, it is the same. Is there a convention? I thought I was using the right convention as for Okta.

Reply:
I updated the placeholders to use the hyphen